actions/push-to-gar-docker/action.yaml

```diff
@@ -28,7 +28,7 @@ inputs:
   file:
     description: |
       The dockerfile to use.
-    default: "Dockerfile"
+    required: true
```

Comment:
doesn't look like this makes it optional?

Reply:
I'm bad at copy-pasting :D